### ISSUE TITLE
extra.name getting set to StudlyCase in composer.json

### ISSUE THREAD

**Issue by brandonkelly** [nystudio107/generator-craftplugin#19]
the `extra.name` property is getting set to a StudlyCase version of the plugin name in the generated composer.json:

```json
"extra": {
  "name": "ElementApi",
```

should be:

```json
"extra": {
  "name": "Element API",
```

**Comment by khalwat** [comment]
Yeah it uses the StudlyCase handle, which is probably wrong. Easy enough to switch it.

**Comment by khalwat** [comment]
Fixed in 1.2.7: https://github.com/nystudio107/generator-craftplugin/releases/tag/1.2.7

Live on https://pluginfactory.io/
